doc/sphinxext/doi_role.py

```diff
@@ -8,12 +8,11 @@
 The link caption will be the raw DOI.
 You can also give an explicit caption, e.g.
 :doi:`Basic local alignment search tool <10.1016/S0022-2836(05)80360-2>`.
-
-:copyright: Copyright 2015  Jon Lund Steffensen. Based on extlinks by
-    the Sphinx team.
-:license: BSD.
 """
 
+# Authors: The scikit-learn developers
```

Comment:
This part was probably copied and pasted from somewhere else (vendored since we talked about it the other day). I guess it should be left as is.